baseline.yaml

```diff
@@ -125,6 +125,38 @@ criteria:
     control_mappings: # TODO
     scorecard_probe:
       - blocksDeleteOnBranches
+  - id: OSPS-TEST
+    maturity_level: 1
+    category: Build & Release
+    criteria: |
+      The project MUST use at least one automated test suite that
+      is publicly released as OSS (this test suite may be maintained
+      as a separate OSS project). It MUST clearly show
+      or document how to run the test suite(s).
+    objective: |
```

Comment:
It's probably worth calling out in the rationale section that this is primarily driven by being able to more quickly and safely resolve vulnerabilities.